src/index.js

```diff
@@ -0,0 +1,21 @@
+import 'core-js/stable';
```

Comment:
I don't think it is really needed, at least I have not added it for data-export. Could you please try without it?

Reply:
Fixed